### PR TITLE
Add LGeneral

### PR DESCRIPTION
**Pull request by Mailaender**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://sourceforge.net/p/lgames/feature-requests/21/
- [X] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)* https://sourceforge.net/p/lgames/patches/31/

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by Mailaender** [comment]
bot, build net.sourceforge.lgames.LGeneral

**Comment by Mailaender** [comment]
bot, build net.sourceforge.lgames.LGeneral

File: net.sourceforge.lgames.LGeneral.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        url: http://downloads.sourceforge.net/lgeneral/lgeneral-1.4.4.tar.gz
+        sha256: 0a26b495716cdcab63b49a294ba31649bc0abe74ce0df48276e52f4a6f323a95
+      - type: file
+        filename: kukgen-data-1.1.tar.gz
+        url: http://downloads.sourceforge.net/lgeneral/kukgen-data-1.1.tar.gz
+        sha256: fbad68e6ec92325ff63dbe6ae77337f9806a4347667c712f94265419b89d6d84
+      - type: file
+        path: net.sourceforge.lgames.LGeneral.metainfo.xml
+      - type: file
+        path: lgeneral128.png
```
Comment (hfiguiere) [inline net.sourceforge.lgames.LGeneral.yaml:32]:
```suggestion
        path: lgeneral128.png
      - type: shell
        commands:
          - cp -p /usr/share/automake-*/config.{sub,guess} .
```

You need something like that to build. (the `.` assume `config.sub` is in the top level and must be adjusted accordingly)

tl;dr: the tarball is so old that the configure bootstrap doesn't know aarch64.

**Comment by Mailaender** [comment]
bot, build net.sourceforge.lgames.LGeneral

**Comment by barthalion** [comment]
bot, build net.sourceforge.lgames.LGeneral

**Comment by barthalion** [comment]
bot, build net.sourceforge.lgames.LGeneral
